yarn-project/acir-simulator/src/avm/avm_context.ts

```diff
@@ -1,140 +1,179 @@
 import { AztecAddress, FunctionSelector } from '@aztec/circuits.js';
 import { Fr } from '@aztec/foundation/fields';
+import { createDebugLogger } from '@aztec/foundation/log';
+
+import { assert } from 'console';
 
 import { AvmExecutionEnvironment } from './avm_execution_environment.js';
-import { AvmMachineState } from './avm_machine_state.js';
-import { AvmMessageCallResult } from './avm_message_call_result.js';
-import { AvmInterpreterError, executeAvm } from './interpreter/index.js';
-import { AvmJournal } from './journal/journal.js';
-import { Instruction } from './opcodes/instruction.js';
+import { AvmMachineState, InitialAvmMachineState } from './avm_machine_state.js';
+import { AvmContractCallResults } from './avm_message_call_result.js';
+import { AvmExecutionError, InvalidProgramCounterError, NoBytecodeForContractError } from './errors.js';
+import { initExecutionEnvironment, initInitialMachineState } from './fixtures/index.js';
```

Comment:
It seems wrong to include test fixture code in non-test code. Do you really need these inits?